nko/index.html

```diff
@@ -552,7 +552,7 @@ <h4>Managing glyph shaping</h4>
 <h3>Context-based shaping &amp; positioning</h3>
 
 
-<p>The  Noto Sans NKo font changes the height of  diacritics according to the height of the base character, but  the preferred approach is to maintain the diacritics at the same height all along a line.<a class="reflink" href="https://github.com/w3c/afrlreq/issues/30" title="GitHub, Issue #30">§</a></p>
+<p>The preferred approach is to maintain the diacritics at the same height all along a line.</p>
```

Comment:
Please don't delete the link to the GH issue.  That's the source for this information.

Reply:
I've added it back, and I also added a `target="_blank"` for the link.